app/routes/meetings/components/MeetingEditor.tsx

```diff
@@ -432,6 +476,31 @@ const MeetingEditor = () => {
                     )}
                   </ConfirmModal>
                 )}
+                {isEditPage &&
+                  canDelete &&
+                  localStorage.getItem(`meeting-${meeting.id}-report`) !==
+                    null && (
+                    <ConfirmModal
+                      title="Hente inn referat fra localstorage?"
+                      message={`Fant ${
+                        localStorage.getItem(`meeting-${meeting.id}-report`) !==
+                        null
+                          ? 1
+                          : 0
+                      } lagret backup for dette møtet. Dette vil overskrive det nåværende referatet lokalt.`}
+                      onConfirm={() => {
+                        loadReportFromLocalStorage(form);
+                      }}
+                      closeOnConfirm
+                    >
+                      {({ openConfirmModal }) => (
+                        <Button danger onClick={openConfirmModal}>
+                          <Icon name="cloud-download-outline"></Icon>
+                          Hent referat lokalt
```

Comment:
```suggestion
                          Hent lokalt referat
```
I feel like this is better wording